pkg/trace/stats/statsraw.go

```diff
@@ -77,14 +87,8 @@ func (s *groupedStats) export(a Aggregation) (pb.ClientGroupedStats, error) {
 }
 
 func newGroupedStats() *groupedStats {
-	okSketch, err := ddsketch.LogCollapsingLowestDenseDDSketch(relativeAccuracy, maxNumBins)
-	if err != nil {
-		log.Errorf("Error when creating ddsketch: %v", err)
-	}
-	errSketch, err := ddsketch.LogCollapsingLowestDenseDDSketch(relativeAccuracy, maxNumBins)
-	if err != nil {
-		log.Errorf("Error when creating ddsketch: %v", err)
-	}
+	okSketch := ddsketch.NewDDSketch(indexMapping, store.NewCollapsingLowestDenseStore(maxNumBins), store.NewCollapsingLowestDenseStore(maxNumBins))
+	errSketch := ddsketch.NewDDSketch(indexMapping, store.NewCollapsingLowestDenseStore(maxNumBins), store.NewCollapsingLowestDenseStore(maxNumBins))
```

Comment:
Why do we not need to catch errors now? Or maybe I should ask why we needed to before 😆

Reply:
So these are errors that can't happen, because gamma is a constant. The errors are thrown if gamma is < 1